tests/phpunit/MergePluginTest.php

```diff
@@ -1026,6 +1028,11 @@ protected function triggerPlugin($package, $directory)
         chdir($directory);
         $this->composer->getPackage()->willReturn($package);
 
+        $event = new \Composer\EventDispatcher\Event(
+            MergePlugin::INIT
+        );
+        $this->fixture->onInit($event);
```

Comment:
There should probably be some tests without the INIT event to so we don't accidentally make some path only work with Composer 1.1+.